### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by tombish**
Discovered by CLion proofreading tool

### CODE REVIEW

**Comment by TheMostDiligent** [comment]
Please use `-DDILIGENT_BUILD_TESTS=TRUE` to build the tests

File: Tests/DiligentCoreAPITest/src/ClearRenderTargetTest.cpp [hunk 1]
```diff
@@ -121,7 +121,7 @@ TEST(ClearRenderTargetTest, AsRenderTarget)
     auto* pEnv       = TestingEnvironment::GetInstance();
     auto* pDevice    = pEnv->GetDevice();
     auto* pSwapChain = pEnv->GetSwapChain();
-    auto* pContext   = pEnv->GetDeviceContext();
```
Comment (TheMostDiligent) [inline Tests/DiligentCoreAPITest/src/ClearRenderTargetTest.cpp]: Here and below pContext is the same, so the second definition should simply be removed

Comment (tombish) [reply Tests/DiligentCoreAPITest/src/ClearRenderTargetTest.cpp]: :grimacing: Sorry made a mess of this should have seen that.

**Comment by TheMostDiligent** [comment]
Linux build is failing because you fixed typo in `GeometryShaderRefenceGL` in CMake, but did not rename the file. Same with `TessellationRefenceGL`

**Comment by tombish** [comment]
Would It be worthwhile starting a new pull request with all changes under one commit, to avoid the messy commit history?

**Pull request merged**